src/Illuminate/Foundation/Application.php

```diff
@@ -1051,7 +1051,95 @@ public function setLocale($locale)
      */
     public function isLocale($locale)
     {
-        return $this->getLocale() == $locale;
+        return $this->getLocale() === $locale;
+    }
+
+    /**
+     * Get default application locale
+     *
+     * @return string
+     */
+    public function getDefaultLocale()
+    {
+        return config('app.fallback_locale');
+    }
+
+    /**
+     * Get additional locales supported by application
+     *
+     * @return array
+     */
+    public function getAdditionalLocales()
+    {
+        return config('app.additional_locales');
+    }
+
+    /**
+     * Get array of all locales supported by application
+     *
+     * @return array
+     */
+    public function getSupportedLocales()
+    {
+        $app_locales = config('app.additional_locales');
+        $app_locales[] = config('app.fallback_locale');
+        return $app_locales;
+    }
+
+    /**
+     * Get application configuration for default locale presence in URL
+     *
+     * @return bool
+     */
+    public function isDefaultLocaleInUrl()
+    {
+        return config('app.default_locale_in_url') === true;
```

Comment:
The rest of the code is using `$this['config']` instead of the `config()` helper. Also, do you really need a method for this?